client/src/App.js

```diff
@@ -0,0 +1,45 @@
+// TODO diese datei kann vermutlich ersetzt/ gelöscht werden --> ermöglicht zunächst aber erstmal die überprüfung, ob der datenbankzugriff steht
+
+import React, { useState, useEffect } from "react";
+
+// SERVICES
+import commentService from './services/commentService';
+
+function App() {
+    const [comments, setcomments] = useState(null);
+
+    useEffect(() => {
+        if(!comments) {
+            getComments();
+        }
+    })
+
+    const getComments = async () => {
+        let res = await commentService.getAll();
+        console.log(res);
+        setcomments(res);
+    }
+
+    const renderComment = comment => {
+        return (
+            <li key={comment._id} className="list__item product">
+                <h3 className="product__name">{comment.autor}</h3>
```

Comment:
@cosmoem ="product__name" better to rewrite it

Reply:
just a small detail

Reply:
i actually decided to leave it like that on purpose when i realised that we'll most likely delete this file anyway 😀 but i can change it

Reply:
actually i just realised this whole file is really not needed at all, i'll just delete it